src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIKnight.java

```diff
@@ -0,0 +1,78 @@
+package com.minecolonies.coremod.entity.ai.citizen.guard;
+
+import com.minecolonies.api.util.constant.ToolType;
+import com.minecolonies.coremod.colony.jobs.JobKnight;
+import com.minecolonies.coremod.entity.ai.util.AIState;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+import static com.minecolonies.coremod.entity.ai.util.AIState.*;
+
+@SuppressWarnings("squid:MaximumInheritanceDepth")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:MaximumInheritanceDepth' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)